pt-BR/mail/chrome/messenger/am-server-advanced.dtd

```diff
@@ -22,7 +22,7 @@
 <!ENTITY overrideNamespaces.label "Permitir que o servidor sobrescreva estes namespaces">
 <!ENTITY overrideNamespaces.accesskey "e">
 <!ENTITY pop3Desc.label "Ao receber mensagens POP deste servidor, salvá-las na:" >
-<!ENTITY folderStorage.label "Where to store mail">
+<!ENTITY folderStorage.label "Onde armazenar emails">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY folderStorage.label "Onde armazenar mensagens">